tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,11 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert not result
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
```

Comment:
Like mentioned in the other testing comment for wave 01, you should test to see if `fatimah`'s inventory has the same 3 items in her inventory after the operation.
```suggestion
    assert not result
    assert len(fatimah.inventory) == 3
    assert len(jolie.inventory) == 0
    assert item_a in fatimah.inventory
    assert item_b in fatimah.inventory
    assert item_c in fatimah.inventory
```